### ISSUE TITLE
install.php and update.php cause an error for a undefined function

### ISSUE THREAD

**Issue by avpaderno** [backdrop/backdrop-issues#1486]
I installed Backdrop, and I kept getting a WSOD because the following error:

> PHP Fatal error:  Call to undefined function config_get() in core/includes/bootstrap.inc on line 1745

I had to edit the install.php file to add the following line.

```
require_once BACKDROP_ROOT . '/core/includes/config.inc';
```

The same happens with update.php.

**Comment by jenlampton** [comment]
I'm seeing the same issue with Backdrop 1.3.2, but at a different line number:
```
Fatal error: Call to undefined function config_get() in backdrop/core/includes/bootstrap.inc on line 1806
```

**Comment by shelane** [comment]
On a brand new site installed with 1.10.0, I attempted to use drush to update and got the error:
```
drush backdrop-pm-update core

		Current Backdrop Version:	1.10.0
		Proposed Backdrop Version:	1.11.2

	Code updates will be made to Backdrop core.

	WARNING: Updating core will discard any
	modifications made to Backdrop core files. If you have made any
	modifications to these files, please back them up before updating so that
	you can re-create your modifications in the updated version of the file.


	Note: Updating core can potentially break your site.
	It is NOT recommended to update production sites without prior testing.


	Do you wish to continue? (y or n): y

Fatal error: Call to undefined function config_get() in /home/docker/.drush/backdrop/commands/pm/backdrop_pm.drush.inc on line 329
Drush command terminated abnormally due to an unrecoverable error.                                                                        [error]
Error: Call to undefined function config_get() in /home/docker/.drush/backdrop/commands/pm/backdrop_pm.drush.inc, line 329
```

**Comment by klonos** [comment]
Thanks for taking the time to report the issue @shelane 👍 

@kiamlaluno @jenlampton do you guys recall what actions were causing this back when you reported this problem? Were you also trying an update? ...was it after issuing drush commands?

**Comment by kiamlaluno** [comment]
I got the error after installing Backdrop, or when trying to run update.php.
I was using Backdrop from the browser. (No Drush command was injured.)

**Comment by klonos** [comment]
@shelane I was able to reproduce the error on a vanilla 1.10.0 using drush, but wondering if this could be a drush-only issue.

@jenlampton @kiamlaluno I had no problems running update.php via the UI. Here's what I did:

1. Install afresh with 1.10.0
2. Install backdrop drush
3. Run `drush backdrop-pm-update core` -> `Call to undefined function config_get() ...` 👎 
4. Add a fake update hook in `system.install`:

  ```
  function system_update_1066() {
    cache_clear_all();
  }
  ```
5. Run update.php -> all good; no errors in the log 👍
